File: VB/DXWebApplication1/Controllers/HomeController.vb
Public Class HomeController
    Inherits System.Web.Mvc.Controller

    Function Index() As ActionResult
        Return View()
    End Function


    Function Designer() As ActionResult
        Dim model As New Models.ReportsModel()
        Return View(model)
    End Function

    Function Viewer() As ActionResult
        Return View()
    End Function
End Class

File: CS/DXWebApplication1/Controllers/HomeController.cs
using System;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace DXWebApplication1.Controllers {
    public class HomeController : Controller {
        public ActionResult Index() {
            return View();
        }

        public ActionResult Designer() {
            Models.ReportsModel model = new Models.ReportsModel();
            return View(model);
        }

        public ActionResult Viewer() {
            return View();
        }
    }
}